### PR TITLE
Fix failing tests.

### PR DESCRIPTION
**Pull request by marcinwasowicz**
Fixes tests failing due to changes introduced in original branch.

### CODE REVIEW

File: test/support/bin/test_bins.ex [hunk 1]
```diff
@@ -220,6 +220,16 @@ defmodule Membrane.Support.Bin.TestBins do
       {{:ok, spec: spec}, state}
     end
 
+    @impl true
+    def handle_notification(
+          %Membrane.Testing.Notification{payload: _payload} = notification,
```
Comment (FelonEkonom) [inline test/support/bin/test_bins.ex:225]: What is the reason of pattern matching, if struct ` %Membrane.Testing.Notification{}` contains field `payload`, if it is enforced key?

File: test/support/bin/test_bins.ex [hunk 2]
```diff
@@ -220,6 +220,16 @@ defmodule Membrane.Support.Bin.TestBins do
...
+          _element,
+          _ctx,
+          state
+        ) do
+      {{:ok, notify: notification}, state}
+    end
+
```
Comment (FelonEkonom) [inline test/support/bin/test_bins.ex:232]: If stopping forwarding notifications in Bins causes fail of some tests, that are not explicitly related to forwarding notifications, I would rather consider the correctness of these tests or maybe unobvious and undesirable behaviors of Bins caused by this change, then just override the changed callback in implementation used in tests

File: test/support/bin/test_bins.ex [hunk 3]
```diff
@@ -220,6 +220,20 @@ defmodule Membrane.Support.Bin.TestBins do
...
+    def handle_notification(_notification, _element, _ctx, state) do
+      {:ok, state}
+    end
```
Comment (andpodob) [inline test/support/bin/test_bins.ex:235]: As we discussed on today's check-in, let's make `test_bin` to forward all notifications.

File: test/support/bin/test_bins.ex [hunk 4]
```diff
@@ -257,6 +271,20 @@ defmodule Membrane.Support.Bin.TestBins do
...
+
```
Comment (andpodob) [inline test/support/bin/test_bins.ex:287]: Same here.

**Review by andpodob — CHANGES_REQUESTED** [review]

**Pull request merged**